apps/intellij/src/main/kotlin/dev/nx/console/generate/NxGenerateRunAnythingProvider.kt

```diff
@@ -0,0 +1,119 @@
+package dev.nx.console.generate
+
+import com.intellij.execution.executors.DefaultDebugExecutor
+import com.intellij.ide.actions.runAnything.RunAnythingAction
+import com.intellij.ide.actions.runAnything.RunAnythingUtil
+import com.intellij.ide.actions.runAnything.activity.RunAnythingCommandLineProvider
+import com.intellij.ide.actions.runAnything.items.RunAnythingItemBase
+import com.intellij.openapi.actionSystem.DataContext
+import com.intellij.openapi.project.Project
+import dev.nx.console.NxIcons
+import dev.nx.console.generate.run_generator.RunGeneratorManager
+import dev.nx.console.models.NxGenerator
+import dev.nx.console.nxls.server.requests.NxGeneratorOptionsRequestOptions
+import dev.nx.console.services.NxlsService
+import javax.swing.Icon
+import kotlinx.coroutines.runBlocking
+
+class NxGenerateRunAnythingProvider : RunAnythingCommandLineProvider() {
+
+    override fun getIcon(value: String): Icon = NxIcons.Action
+
+    override fun getHelpGroupTitle() = "Nx"
+    override fun execute(dataContext: DataContext, value: String) {
+        super.execute(dataContext, value)
+    }
+
```

Comment:
How about adding additional info with `getAdText()` here. To let users better understand that they can use `shift + enter` for a dry run, we could have it as a grey text in the bottom.

Reply:
Yes a good idea, 
a text could be "Press `shift` to run with dryRun mode enabled"
<img width="716" alt="Screenshot 2023-02-28 at 09 55 50" src="https://user-images.githubusercontent.com/9085108/221802831-626d1413-924d-44f6-b8c4-999a02eb8146.png">